_pages/about.md

```diff
@@ -11,35 +11,39 @@ modified: 2016-04-18T16:39:37-04:00
 | Decide to work with Lantern | Great! You've had a clear piece of research in mind, but you don't just want to write another article which will gather dust. Instead, you want your work to contribute to a much larger project where it can feed into a collaborative systematic search to gain true economic insight. |
-| Check you've got the right skills | Make sure that you have at least basic programming skills in Lantern's main programming language. | 
-| Review the current Lantern Framework | The Lantern Framework is the scaffolding on which the Global Economic Model is built. This effectively breaks down the economy into different sectors (i.e., GDP prediction, Capital Markets and Flows, International trade, etc...) and maps out how these sectors communicate and how economic data flows from one sector to another within the model. The proposed Lantern project is designed to be highly modular, meaning that you can translate your research into a small sub-model which fits into the overall framework. |      
-| Submit a proposal | You've reviewed the framework and can see a missing piece in the jigsaw, so it's time to fill in the online form to propose a project. Your project will then be assessed by our panel 1) to make sure that a similar project does not already exist, 2) check that the key assumptions in your model are defensible (i.e. not clearly contradicted by real world evidence) 3) check that it is testable (i.e. that it makes predictions using readily available macroeconomic data and so can be tested on historic data and on future performance) and 4) to give it an overall score to assess its potential usefulness. |  
+| Check you've got the right skills | Make sure that you have at least basic programming skills in Lantern's main programming language Python. | 
+| Review the current Lantern Framework | The Lantern Framework is the scaffolding on which the *global economic model* is built. This effectively breaks down the economy into different sectors (i.e., GDP prediction, Capital Markets and Flows, International trade, etc...) and maps out how these sectors communicate and how economic data flows from one sector to another within the model. The proposed Lantern project is designed to be highly modular, meaning that you can translate your research into a small sub-model which fits into the overall framework. |      
+| Submit a proposal | You've reviewed the framework and can see a missing piece, so it's time to fill in the online form to propose a project. Your project will then be assessed by our panel to 
+1) to make sure that a similar project does not already exist 
+2) check that the key assumptions in your model are defensible (i.e. not clearly contradicted by real world evidence) 
+3) check that it is testable (i.e. that it makes predictions using readily available macroeconomic data and so can be tested on historic data and on future performance) and 
+4) to give it an overall score to assess its potential usefulness. |  
```

Comment:
Added each value on a new line, might be better to use an actual markdown list that auto formats when you use `1.` instead of `1)`